src/FormulateInput.vue

```diff
@@ -100,9 +100,15 @@ export default {
     }
   },
   inject: {
+    // For some reason formulateSetter always returns undefined if given a default,
+    // even when it really should have been provided 
     formulateSetter: { default: undefined },
+    formulateSetter: 'formulateSetter',
     formulateFieldValidation: { default: () => () => ({}) },
-    formulateRegister: { default: undefined },
+    // For some reason formulateRegister always returns undefined if given a default,
+    // even when it really should have been provided 
+    formulateRegister: { default: () => undefined },
+    formulateRegister: 'formulateRegister',
```

Comment:
This felt very much like a bug, but I'd have to check the provide/inject stuff in Vue 3 in isolation a bit more to check if anything has changed here in their API. A glance through their documentation doesn't suggest this should have changed.